addons/mail/static/src/xml/thread.xml

```diff
@@ -20,10 +20,16 @@
                     <i class="fa fa-picture-o mr8" t-if="widget.activeAttachment.fileType == 'image'" role="img" aria-label="Image" title="Image"/>
                     <i class="fa fa-file-text mr8" t-if="widget.activeAttachment.fileType == 'application/pdf'" role="img" aria-label="PDF file" title="PDF file"/>
                     <i class="fa fa-video-camera mr8" t-if="widget.activeAttachment.fileType == 'video'" role="img" aria-label="Video" title="Video"/>
-                    <t t-esc="widget.activeAttachment.name"/>
-                    <a role="button" href="#" class="o_download_btn ml8 small" data-toggle="tooltip" data-placement="right" title="Download"><i class="fa fa-fw fa-download" role="img" aria-label="Download"/></a>
+                    <span class="o_viewer_document_name" t-esc="widget.activeAttachment.name"/>
                 </span>
-                <a role="button" class="o_close_btn float-right" href="#" aria-label="Close" title="Close">×</a>
+                <div class="o-autogrow"/>
+                <button class="btn btn-sm btn-primary o_download_btn" data-toggle="tooltip">
+                    <i class="fa fa-fw fa-download" role="img" aria-label="Download"/>
+                    <span class="d-none d-md-inline">Download</span>
+                </button>
+                <a role="button" href="#" class="btn btn-sm btn-secondary o_close_btn" title="Close">
+                    <i class="fa fa-fw fa-close" role="img" aria-label="Close"/>
+                </a>
```

Comment:
Same remark as before, this layout is deprecated. The DocumentViewer widget is deprecated version of AttachmentViewer, which we want to convert ASAP.